authors/apps/profiles/urls.py

```diff
@@ -12,5 +17,8 @@
     path("follow/following", ProfileFollowingAPIView.as_view(),
          name="following"),
     path("follow/followers", ProfileFollowersAPIView.as_view(),
-         name="followers")
+         name="followers"),
+    path("", UserProfileListAPIView.as_view(), name="profile-list"),
+    path("<username>", ProfileRetrieveUpdateView.as_view(),
```

Comment:
why are we having two exact urls achieving the same thing

Reply:
Thanks for the feedback @missvicki but please pay close attention to this because these two URLs are doing different things
Thanks again.

Reply:
Worked on